### DISCUSSION TITLE
Trying to add Kermit to my KMM app but get Unresolved reference: Logger

### DISCUSSION THREAD

**Discussion by antranapp** [touchlab/Kermit#221] [Q&A] [answered]
Thanks for the great lib

I'm trying to add this lib into my KMM app. Following the readme, I added this 2 lines into my `commonMain` dependencies

```
val commonMain by getting {
        dependencies {
           implementation(kotlin("stdlib-common"))
           implementation("co.touchlab:kermit:$kermitVersion")
      }
}
```
After Gradle sync, Logger is still not resolvable. Any other step I have to do?

This is what Android Studio tell me to do

![Screenshot 2022-01-23 at 7 19 53 AM](https://user-images.githubusercontent.com/478757/150658547-12400505-ccef-4c8b-9769-3b2bcaed071d.png)

(sorry for nope question, I'm an iOS Dev exploring KMM)

**Comment by kpgalligan** [comment, marked as answer]
It's some kind of config issue. It's difficult to say without looking at the rest of your config. If this is something public I can take a look. If not, hmm. It's more difficult to figure out in the abstract.

Can you post gradle configs? So, the full gradle file for the shared module. Also some other questions:
- What version of Kotlin are you on?
- Where is this source? Is it in the shared module or in the Android source?

You may be able to try running the build from the command line. Sometimes the IDE can't resolve something but the build works.

Generally speaking, though, gradle config issues need more info as its tough to debug without that context.

**Reply by antranapp** [reply]
Thank you for the direction. I'll take a look again.
